internal/server/http/handler/v1/tree/handler.go

```diff
@@ -140,12 +140,21 @@ func UpdateTree(_ service.TreeService) fiber.Handler {
 // @Failure		403	{object}	HTTPError
 // @Failure		404	{object}	HTTPError
 // @Failure		500	{object}	HTTPError
-// @Router			/v1/tree [delete]
+// @Router			/v1/tree/{tree_id} [delete]
+// @Param			tree_id			query	string	false	"Tree ID"
```

Comment:
It's not a query parameter, it's a path parameter.

```suggestion
// @Param			tree_id			path	string	false	"Tree ID"
```